include/glow/Backends/CompiledFunction.h

```diff
@@ -60,7 +60,16 @@ class CompiledFunction {
   virtual ~CompiledFunction() = default;
   /// Execute the network and allocate Placeholder memory with given
   /// \p ctx providing mapping between Placeholder and populated tensor.
-  virtual void execute(Context &ctx) = 0;
+  virtual void execute() = 0;
+
+  /// Does any needed initialization work for the Backend.
```

Comment:
can you add brief explanation of potential actions for each method. Copy constants, init device etc.